react-app/src/components/property/PropertyTable.tsx

```diff
@@ -307,11 +307,11 @@ const PropertyTable = (props: IPropertyTable) => {
           customExcelMap={excelDataMap}
           columns={columns}
           addTooltip="Create New Property"
-          initialState={{
-            sorting: {
-              sortModel: [{ sort: 'desc', field: 'UpdatedOn' }],
-            },
-          }}
+          // initialState={{
+          //   sorting: {
+          //     sortModel: [{ sort: 'desc', field: 'UpdatedOn' }],
+          //   },
+          // }}
```

Comment:
It's too bad this can't still be set within the table itself. I see it included in the routes, but if you don't come from the routes but also have no query params, will there be an initial state?